tests/test_cli.py

```diff
@@ -52,6 +53,29 @@ def objective_func_multi_objective(trial: Trial) -> Tuple[float, float]:
     return (x + 5) ** 2, (x - 5) ** 2
 
 
+def _parse_output(output: str, output_format: str) -> Any:
```

Comment:
The return type of this function is `Any` due to `yamls.load` and `json.load`, but I want to know what kind of data (A list of line contents in the table. A corresponding list is returned if the format type is `json` and `yaml`) is returned from this function. Could you add a code comment about it?